cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -68,12 +68,17 @@
     /sys/fs/cgroup/freezer/snap.*/ w,
     /sys/fs/cgroup/freezer/snap.*/cgroup.procs rw,
 
-    # cgroup: pids
+    # cgroup: snapd
+    # allow creating and configuring the snapd hierarchy
+    /run/snapd/ rw,
+    /run/snapd/cgroup/ rw,
+    mount fstype=cgroup options=(rw nosuid nodev noexec relatime) cgroup -> /run/snapd/cgroup/,
+    /run/snapd/cgroup/release_agent rw,
+    /run/snapd/cgroup/notify_on_release rw,
```

Comment:
I understand we are going to use those files in the next PR but we don't need this yet.

Reply:
Yeah but I want to make it clear this is going to be needed. If we are going to get a security review we might as well do it for the full set of permissions.

Reply:
Won't we need another security review for that PR anyways? If so, I'd say just do it then when it will be clear what these files will be used for and why we need rw for those files.

Reply:
I think it's small enough to keep it here. This allows me to introduce the agent as a standalone entity and simply wire it here without additional security review for this part. It's exactly two lines and the purpose is clear.

Reply:
I'm ok with the files being added here, but it isn't clear if both 'rw' is needed or just one without the other code.

Reply:
Did you have a chance to double check this?